.github/workflows/pre-release.yml

```
@@ -36,9 +36,26 @@ jobs:
 
       - name: Enter RC mode (if not already in RC mode)
         run: |
-          # ensure we’re in the right pre-mode (tag "rc")
-          if [ ! -f .changeset/pre.json ] \
-            || [ "$(jq -r '.tag' .changeset/pre.json 2>/dev/null || echo '')" != "rc" ]; then
+          # Check if we're in pre-release mode with the "rc" tag
+          if [ -f .changeset/pre.json ]; then
+            MODE=$(jq -r '.mode' .changeset/pre.json 2>/dev/null || echo '')
+            TAG=$(jq -r '.tag' .changeset/pre.json 2>/dev/null || echo '')
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)